src/api/sip.ts

```diff
@@ -191,24 +220,24 @@ export class SIP extends EventEmitter {
       '/JsSIP ' +
       jssipVersion
     //
-    const callOptions = ((sipLoginOption.pbxTurnEnabled && turnConfig) ||
-      {}) as CallOptions
-    if (!callOptions.pcConfig) {
-      callOptions.pcConfig = {}
+    const callOptions = ((pbxTurnEnabled && turnConfig) || {}) as CallOptions
+    let { pcConfig } = callOptions
```

Comment:
you have to copy main object, 
ex. `const {pcConfig} = callOptions
 let abc  = {...pcConfig}`